src/main/java/configuration/InterceptorConfig.java

```diff
@@ -0,0 +1,34 @@
+package configuration;
```

Comment:
package names look incorrect on a few classes

Reply:
Changed package names and the logs are now appearing.